### ISSUE TITLE
Replace esbuild for Vite as build tool

### ISSUE THREAD

**Issue by henriquehbr** [henriquehbr/svelte-typewriter#38]
> Part of the maintenance mode [roadmap](https://github.com/henriquehbr/svelte-typewriter/projects/4) (#35)

This one is a speculation at the moment, i'm studying the possibilities of adding it as a replacement for esbuild, the killer features that convinced me to start cogitating such change is the almost instant startup offered by their dev server and the fact that Snowpack uses esbuild under the hood, which makes it really, really fast (among many others benefits out-of-the-box)

**Comment by henriquehbr** [comment]
### Update

**tl;dr:** Instead of migrating to Snowpack, Vite will be the new build tool used in `svelte-typewriter`

As officially announced by Evan You on [this article](https://dev.to/yyx990803/announcing-vite-2-0-2f0a), Vite 2.0 is out, with a bunch of improvements, and one of them in particular that called my attention, was the idea of migrating to a more "framework-agnostic" core, this means that Vite now works with any framework out there, not only for Vue SFC's anymore, besides that, there's also a whole bunch of other improvements that Vite offers over it's competitor that makes it a more interesting choice, there's a whole section specifically about it [on the docs](https://vitejs.dev/guide/comparisons.html#snowpack)

> **PS:** this "dev-only feature" was requested on #45, and it's being work on as a top priority at the moment

**Comment by henriquehbr** [comment]
Special thanks to @alesvaupotic, for his [svelte-routify-windi-vite](https://github.com/reepolee/svelte-routify-windi-vite) template which was of huge help in the integration of Svelte with Vite

**Comment by alesvaupotic** [comment]
Oh, thanks for the compliments! I give all the credit to the authors of those packages, I just glued it all together. 

Wish you all the best with the project!
